src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/services/SearchFunctionalityITCase.java

```diff
@@ -84,4 +84,31 @@ public void testSearchWithBadDescription() {
         else
             assert false;
     }
+    
+    @Test
+    public void testSearchWithEmptyString() {
+        if (search("", String.valueOf(UUID.randomUUID().toString())) == null) {
+            assert true;
+        } else {
+            assert false;
+        }
+    }
+
+    @Test
+    public void testSearchWithDoubleQuotes() {
+        if (search("\"", String.valueOf(UUID.randomUUID().toString())) == null) {
```

Comment:
@vladbenchea Is this correct? The declaration for search is:
protected EntityModel search(String searchField, String query)
so shouldn't it be search("name", "\"")?
Also you would need to create an actual entity with a quote in the name

Reply:
I just copied how the other tests were made in that class. 
Also, you don't really need an entity with quotes in name because this tests if it can search for that specific character